app/models/groups_metadata_delegate_regions.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class GroupsMetadataDelegateRegions < ApplicationRecord
+  has_one :user_group, -> { where(metadata_type: 'GroupsMetadataDelegateRegions') }, class_name: 'UserGroup', foreign_key: 'metadata_id'
```

Comment:
This is not how polymorphic associations work in Rails.
There should be a `belongs_to` in UserGroup, use the name of that association with the `as:` keyword here.

Further reading: https://guides.rubyonrails.org/association_basics.html#polymorphic-associations

Reply:
Oh that's shorter. Thanks. Changed in groups_metadata_teams_committees.rb as well in this PR, hope it's fine.